platform/android/java/lib/src/org/godotengine/godot/input/GodotEditText.java

```diff
@@ -101,7 +113,11 @@ protected void initView() {
 	}
 
 	public boolean isMultiline() {
-		return mMultiline;
+		return mKeyboardType == VirtualKeyboardType.KEYBOARD_TYPE_MULTILINE;
+	}
```

Comment:
I agree with @akien-mga; let's remove this for the `master` branch.

Reply:
Done